build/manifest.ps1

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env pwsh
-#Requires -PSEdition Core
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Good idea to add this header as well; can we keep the old shebang along with this header?

Reply:
Yes, thanks for explaining this offline! Restoring both of these lines so that we'll continue to use PowerShell Core for this script.